### PR TITLE
169 coordinates to ink

### LINKED ISSUE #169: Convert coordinates to ink objects

### PR DESCRIPTION
**Pull request by clementcharmillot**
closes #169

### CODE REVIEW

File: app/src/androidTest/java/com/epfl/drawyourpath/utils/UtilsTest.kt
Comment (Gafragni) [inline app/src/androidTest/java/com/epfl/drawyourpath/utils/UtilsTest.kt:1]: Thanks for adding and implementing a test class for utils, we will need it quit a bit this sprint

File: app/src/main/java/com/epfl/drawyourpath/utils/Utils.kt [hunk 1]
```diff
@@ -223,4 +228,32 @@ object Utils {
...
+     * Converts a LatLng to a Point.
+     * @param coordinate The coordinate we want to convert
+     * @return A Point object representing the coordinate in planar space.
+     */
+    fun coordinateToPoint(coordinate: LatLng): Point {
+        val lat = Math.toRadians(coordinate.latitude)
+        val long = Math.toRadians(coordinate.longitude)
+        // Mercator projection formula
+        val y = ln(tan(lat) + (1 / cos(lat)))
+        return Point.create(long.toFloat(), y.toFloat())
```
Comment (Gafragni) [inline app/src/main/java/com/epfl/drawyourpath/utils/Utils.kt:257]: Neat formula for the Mercator projection !
There are many ways to project but it is probably the best one for this application, as this is the most common projection and it is not too expensive to compute.

**Review by Gafragni — APPROVED** [review]
Well done !
Some simple and readable code that is fully tested, LGTM

File: app/src/main/java/com/epfl/drawyourpath/utils/Utils.kt (hunk 1 above)
Comment (clementcharmillot) [reply app/src/main/java/com/epfl/drawyourpath/utils/Utils.kt:257]: Yes, it is very cheap. But it applies some stretching around the poles. To see if it poses a problem for later.

**Pull request merged**